addons/l10n_it_edi/views/l10n_it_view.xml

```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <record id="res_partner_tree_l10n_it" model="ir.ui.view">
```

Comment:
Should not this be used somewhere?

Reply:
It's not primary, so it modifies the view everywhere.
Although they should not be visible if we aren't using an Italian company 🤔

Reply:
The mode below is primary

Reply:
😅 you're right, sorry.
I'll investigate... we should add these field to be able to correct them when clicking the link from the wizard